app/helpers/admin/users_helper.rb

```diff
@@ -0,0 +1,21 @@
+module Admin::UsersHelper
+  def get_gender(gender)
+    gender == 'M' ? 'Male' : 'Female'
+  end
+  
+  def get_enable_or_disable_link(user)
+    if user.enabled?
+      link_to 'Disable', enable_or_disable_user_admin_user_path(user), data: { toggle: "tooltip" }, title: 'Disable User', remote: true
+    else
+      link_to 'Enable', enable_or_disable_user_admin_user_path(user), data: { toggle: "tooltip" }, title: 'Enable User', remote: true
+    end
+  end
+  
+  def promote_to_manager_or_demote_to_user_link(user)
+    if user.manager?
```

Comment:
not sure, i guess something similar to what was suggested above could work here.

Reply:
its just an idea, hoping you can come with something even better in this case or future cases.

Reply:
Above suggested idea worked.

Reply:
This worked: 
role = user.manager?  ? 'Demote to' : 'Promote to'

link_to role + ' User', update_role_admin_user_path(user), data: { toggle: "tooltip" }, title: '#{role} User', remote: true